.github/workflows/macos.yml

```diff
@@ -329,6 +333,8 @@ jobs:
           PYTEST_ADDOPTS: '--color=yes'
           PYTEST_XDIST_AUTO_NUM_WORKERS: 4
         run: |
+          brew services start caddy
+          brew services start vsftpd
```

Comment:
neither this

Reply:
Indeed. Deleted them.

But, enabling them and the 27 extra tests with them causes a 12x time increase for pytests:

With caddy and vsftpd disabled/undetected:
`====================== 437 passed, 303 skipped in 41.64s =======================`
https://github.com/curl/curl/actions/runs/15067273333/job/42354915109#step:17:1535

Detected/configured + with `brew services start`:
`================= 464 passed, 276 skipped in 490.95s (0:08:10) =================`
https://github.com/curl/curl/actions/runs/15067659060/job/42356063606?pr=17362#step:17:1542

Detected/configured + without `brew services start`:
`================= 464 passed, 276 skipped in 487.64s (0:08:07) =================`
https://github.com/curl/curl/actions/runs/15068207198/job/42357850173?pr=17362#step:17:1537

Reply:
Interesting. Well, then we enable them...but I have no idea why it makes a difference.